src/pages/Facility/settings/locations/LocationForm.tsx

```diff
@@ -223,7 +223,7 @@ export default function LocationForm({
                       <SelectValue />
                     </SelectTrigger>
                   </FormControl>
-                  <SelectContent>
+                  <SelectContent className="max-h-[20rem]">
```

Comment:
Let's do viewport relative height, rather than fixed (something like `max-h-[40vh]` test and see what works).